setup.py

```diff
@@ -60,7 +61,7 @@
         "click>=8.1.0,<8.2",
         "hexbytes>=0.2.2,<0.3",
         "pydantic>=1.9.0,<2.0",
-        "eth-ape>=0.2.8,<0.3.0",
+        "eth-ape==0.2.8",
```

Comment:
upgrading to ape 0.3.0 is going to be challenge but realizing now I didn't need to make this change
```suggestion
        "eth-ape>=0.2.8,<0.3.0",
```